dev/arktype.io/docs/api/arrayof.md

```diff
@@ -4,8 +4,24 @@ hide_table_of_contents: true
 
 # arrayOf
 
-## text
+## operator
 
-```ts
-arrayOf: Ark["arrayOf"]
-```
+-   [arrayOf](./arrayof.md)
+
+## string
+
+-   "type[]"
+
+## tuple
+
+-   ["arrayOf", &lt;object&gt;]
+
+## helper
+
+-   arrayOf(&lt;object&gt;)
+
+## example
+
+-   const arrayOf = type("number[]") <br/>
+-   const arrayOf = type(["arrayOf", "number"]) <br/>
+-   const arrayOf = arrayOf("number") <br/>
```

Comment:
Change variable name, this would overwrite itself. Maybe something like `numberArray` for a case like this